src/connect/wrapMapToProps.ts

```diff
@@ -20,7 +20,13 @@ export function wrapMapToPropsConstant(
   //  could be a dispatch function in some cases (ex: whenMapDispatchToPropsIsMissing)
   //  and a state object in some others (ex: whenMapStateToPropsIsMissing)
   // eslint-disable-next-line no-unused-vars
-  getConstant: (dispatch: Dispatch) => { dispatch?: Dispatch }
+  getConstant: (dispatch: Dispatch) =>
```

Comment:
:hand: This should probably be reworked to be more generic and flexible - maybe like:

`getConstant: <T>(value: T) => ( () => T)`

In other words, whatever you pass in, it returns a function that returns that thing.

Reply:
It doesn't match with the usage of `wrapMapToPropsConstant` made by `whenMapDispatchToPropsIsMissing`:

```ts
 wrapMapToPropsConstant((dispatch: Dispatch) => ({
      dispatch,
 }))
```

Reply:
Good point, it's not just a passthrough